### ISSUE TITLE
AttributeError: 'Tensor' object has no attribute 'true_divide_'

### ISSUE THREAD

**Issue by KeepGoing2019HaHa** [open-mmlab/mmdetection#4046]
Hi, I was trying to run vfnet config, but an error raised, 

```
>   File "/data/work/mmdetection/mmdet/models/dense_heads/vfnet_head.py", line 403, in loss
>     pos_inds.new_tensor(num_pos).float()).item()
>   File "/data/work/mmdetection/mmdet/models/dense_heads/vfnet_head.py", line 24, in reduce_mean
>     tensor.true_divide_(tensor.new_tensor(dist.get_world_size())),
> AttributeError: 'Tensor' object has no attribute 'true_divide_'
```

package version is :
mmdet: latest version
pytroch: 1.3.1
mmcv: 1.1.6

Can someone help me? thanks in advance.

**Comment by yuzhj** [comment]
the version of pytorch is too old, upgraded to version 1.6 can solve this problem

**Comment by ZwwWayne** [comment]
This should be an incompatibility bug. We will fix it to make it compatible with Pytorch 1.3 until Pytorch 1.3 is no longer supported.
